helpers/issuesFilter.js

```diff
@@ -0,0 +1,71 @@
+const { prefixes, checkSomeLabelEquals, checkSomeLabelHasPrefix } = require('../constants');
+
+async function fetchIssuesInMilestoneByFilters (octokit, params) {
+    const { customer, includeIssueForAll } = params;
+
+    const filterIssuesForCustomer = function (issues) {
+      return issues.filter( issue => checkSomeLabelEquals(issue.labels, prefixes.CUSTOMER, customer));
+    }
+
+    const filterIssuesForAnyCustomer = function (issues) {
+      return issues.filter( issue => !checkSomeLabelHasPrefix(issue.labels, prefixes.CUSTOMER));
+    }
+
+    const issues = await octokit.getIssues(params);
+
+    if(customer && !includeIssueForAll) {
+      return { forCustomer: filterIssuesForCustomer(issues) };
+    }
+    else if(customer){
+      return {
+        forCustomer: filterIssuesForCustomer(issues),
+        noCustomer: filterIssuesForAnyCustomer(issues)
+      };
+    }
+    else return { noCustomer: issues };
+}
+
+async function fetchIssuesByFilter(octokit, params, group){
+  params.filter = group;
+  return await fetchIssuesInMilestoneByFilters(octokit, params);
```

Comment:
This is mutating parameters :cry: . Best to clone them. Nice way to do this is with spread syntax:

```suggestion
  const issueParams = { ...params, filter: group };
  return await fetchIssuesInMilestoneByFilters(octokit, issueParams);
```